### PR TITLE
Bugfix trigger and samples

### PR DESCRIPTION
**Pull request by abolshov**
quickly fix some bugs

### CODE REVIEW

**Comment by ahmad3213** [comment]
One commit here which is adding generator in config/Run3_2022/samples.yaml
is also in my PRs ( which is also mot merged)

File: config/HH_bbWW/Run3_2022/samples.yaml [hunk 1]
```diff
@@ -553,55 +555,55 @@ VBFHHto2B2V_CV_1_C2V_0_C3_1:
...
 VBFHHto2B2Vto2L2Nu_CV-m0p012_C2V-0p030_C3-10p2:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
 VBFHHto2B2Vto2L2Nu_CV-m0p758_C2V-1p44_C3-m19p3:
   sampleType: HHnonRes
   generator: madgraph
-  #crossSection: To_be_added  
+  #crossSection: To_be_added
```
Comment (kandrosov) [inline config/HH_bbWW/Run3_2022/samples.yaml:606]: AnaTuple production will not run without crossSection value. It is expected to be fixed in #63.

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Comment by ahmad3213** [comment]
This PR can be closed, all changes here either fixed by #63 or #65

**Comment by kandrosov** [comment]
- empty triggers case handled in #65
- missing generator entries in #63
closing this PR

**Pull request closed without merging**